src/language-css/printer-postcss.js

```diff
@@ -493,8 +493,8 @@ function genericPrint(path, options, print) {
         grandParent.type === "value-func" &&
         grandParent.value === "selector"
       ) {
-        const start = locStart(parentNode.open) + 1;
-        const end = locEnd(parentNode.close) - 1;
+        const start = locEnd(parentNode.open) + 1;
```

Comment:
This wasn't really necessary but I think it makes sense to align `locEnd(open)` and `locStart(close)`